app/models/company.rb

```diff
@@ -26,4 +28,22 @@ def self.company_count_by_status(status)
   def self.company_count_by_city
     group(:city).count.sort_by { |_key, value| value }.pop(10).reverse
   end
+
+  private
+
+  def create_new_user
+    password = User.generate_password
+    company = Company.last
+    user = User.create(email: company.email,
+                       name: company.name,
+                       title: company.title,
+                       password: password,
+                       company_id: company.id
+                       )
+    email_new_user(user, password)
```

Comment:
What happens if there is an error and the user is not created? Will it send an email anyway? or will the app break? Do we need to have some type of error handling here, perhaps an "if user.save" then send email, if not, flash message or redirect somewhere? Or are we catching errors before this point so the user will always be created?